pkgs/applications/misc/spice-autorandr/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib, stdenv, fetchFromGitHub, pkg-config, autoconf, automake, libX11, libXrandr }:
+stdenv.mkDerivation rec {
+  name = "spice-autorandr";
+  version = "0.0.2";
+
+  src = fetchFromGitHub {
+    owner = "seife";
+    repo = name;
+    rev = "0f61dc9";
+    sha256 = "sha256-eBvzalWT3xI8+uNns0/ZyRes91ePpj0beKb8UBVqo0E=";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ autoconf automake libX11 libXrandr ];
+  buildPhase = ''
+    autoreconf -is
+    ./configure
+    make
+  '';
```

Comment:
This looks very much like the normal buildphase. Doe it not work when you just omit `buildPhase` here ?

Reply:
You are right, it works when I omit `buildPhase` 👍

Reply:
After further testing I realized that I was still testing the previous version of the package with `buildPhase` still included. After removing `buildPhase` the build eventually failed during `installPhase` like so:

```
❯ nix-build -A spice-autorandr
this derivation will be built:
  /nix/store/jhkrw793y0i52h037yip6b0lj5b2kqwr-spice-autorandr.drv
building '/nix/store/jhkrw793y0i52h037yip6b0lj5b2kqwr-spice-autorandr.drv'...
unpacking sources
unpacking source archive /nix/store/06dq89f8lip7z46ibq2dx861iipjk5bb-source
source root is source
patching sources
updateAutotoolsGnuConfigScriptsPhase
configuring
no configure script, doing nothing
building
no Makefile or custom buildPhase, doing nothing
installing
cp: cannot stat 'spice-autorandr': No such file or directory
```

I corrected this by adding

```
  preConfigure = ''
    autoreconf -is
  '';
```